api/register.go

```diff
@@ -0,0 +1,60 @@
+package api
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	k8scheme "k8s.io/client-go/kubernetes/scheme"
+)
+
+const (
+	// GroupName is the group name use in this package
+	GroupName = "alpha.caicloud.io"
```

Comment:
nit: s/GroupName/AlphaGroupName?

When this graduate to beta, how do you handle compatibility?  You can't just change GroupName to "beta.caicloud.io", this will break things.